src/main/java/me/zikani/labs/pakadali/api/qr/validator/impl/QRCodeValidatorImpl.java

```diff
@@ -0,0 +1,19 @@
+package me.zikani.labs.pakadali.api.qr.validator.impl;
+
+import me.zikani.labs.pakadali.api.qr.validator.interfaces.QRCodeValidator;
+
+public class QRCodeValidatorImpl implements QRCodeValidator {
+    private static final int QR_MAX_SIZE = 9999;
+    public boolean isQRCodeSizeValid(String size)
+    {
+        size = size.trim();
```

Comment:
Perhaps add a null check on `size` for posterity.

Reply:
Null checking would useful...will work on it